src/App.jsx

```diff
@@ -45,6 +46,7 @@ function App() {
             <Route path="/user" element={<h1>USER PAGE</h1>} />
             <Route path="/favorites" element={<h1>FAVORITES</h1>} />
             <Route path="/sell-house" element={<SellHousePage />} />
+            <Route path="/map" element={<GoogleMapComp />} />
```

Comment:
It's a good idea to have a separate page for the map only but for now it might be easier to keep the map only as a part of other components.